packages/fluentui/react-northstar/src/themes/teams/getBorderFocusStyles.ts

```diff
@@ -51,31 +32,42 @@ export const getBorderFocusStyles = (args: BorderFocusStyles): ICSSInJSStyle =>
     focusOuterBorderColor = sv.focusOuterBorderColor || defaultColor,
     borderPadding,
   } = args;
-
-  const afterBorderEdgeValue =
-    borderPadding == null ? `-${borderWidth}` : `calc(0px - ${borderPadding} - ${borderWidth})`;
-  const beforeBorderEdgeValue = borderPadding == null ? '0' : `-${borderPadding}`;
-
+  const defaultPreudoStyles: React.CSSProperties = {
+    content: '""',
+    position: 'absolute',
+    borderStyle: 'solid',
+    pointerEvents: 'none',
+    borderWidth,
+    borderRadius,
+  };
+  const borderPaddingTop = (borderPadding as BorderPadding)?.top || borderPadding;
+  const borderPaddingBottom = (borderPadding as BorderPadding)?.bottom || borderPadding;
+  const borderPaddingLeft = (borderPadding as BorderPadding)?.left || borderPadding;
+  const borderPaddingRight = (borderPadding as BorderPadding)?.right || borderPadding;
   return {
     ':focus': {
-      outline: 'none',
+      outline: 0,
```

Comment:
```suggestion
      outline: 'none',
```

This is falsy change